engine/release-notes/23.0.md

```diff
@@ -81,6 +81,7 @@ For a full list of pull requests and changes in this release, refer to the relev
 - API: Add a `CreateMountPoint` parameter to `POST /containers/create`. [moby/moby#43484](https://github.com/moby/moby/pull/43484)
 - API: Add a `shared-size` parameter to `GET /images/json` to enable shared-size computation of images. [moby/moby#42531](https://github.com/moby/moby/pull/42531)
 - API: Add a `type` parameter to `GET /system/df`, to control what object types to are considered when computing disk usage. [moby/moby#42559](https://github.com/moby/moby/pull/42559)
+- systemd: Use a systemd-managed containerd instead of a daemon-managed containerd. [moby/moby#42373](https://github.com/moby/moby/pull/42373)
```

Comment:
I guess, same for this (as we used a separate containerd service since .. (long ago)

Reply:
I think this is correct, as it is now impossible for the daemon to start a managed containerd when running under systemd.